src/items/type-aliases.md

```diff
@@ -37,6 +37,9 @@ A type alias without the [_Type_] specification may only appear as an
 A type alias with [_TypeParamBounds_] may only specified when used as
 an [associated type] in a [trait].
 
+A type alias with where clauses after the equals sign may only appear as an
+[associated type] in a [trait] or a [trait impl].
```

Comment:
Hm, so I may be a little confused here. IIUC, `associated_type_defaults` is not being stabilized? If so, this seems to imply that `=` can be used in a trait definition. Due to #24 (type defaults are not documented at all), it makes it a bit awkward to talk about this. 

If I am understanding this correctly, I'm not sure how to handle it. Perhaps add a sentence that type alias defaults are not allowed in trait definitions?

Reply:
Yeah, oops. This is unnecessary.